docs/releases/1.10.txt

```diff
@@ -838,7 +838,7 @@ Using ``request.user.is_authenticated()`` and ``request.user.is_anonymous()`` as
 The following members of
 :class:`~django.contrib.auth.models.AbstractBaseUser` and
 :class:`~django.contrib.auth.models.AnonymousUser` classes are now
-properties, and should no longer be invoked as methods:
+properties, and should no longer be invoked as methods::
```

Comment:
Is this change needed? Since the next block is not code, single colon seems more natural to me.